### PR TITLE
feature: allow configurable title

### LINKED ISSUE #15: Customizable title
Originally described by PDXUser. 

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
How about a customize-able page header?  Something where we could input in the config file or in the web interface.  I can envision something like "Server Command Center" or "Server Control"

Notes; 

- Probably something for GetButtons, after entity support is implemented.
- This would go in the config file

### PR DESCRIPTION
**Pull request by strazto**
Resolves #15 

# Checklist
Please put a X in the boxes as evidence of reading through the checklist.

- [x] I have forked the project, and raised this PR on a feature branch. 
- [ ] `make githooks` has been run, and my git commit message was accepted by the git hook.
- [ ] `make daemon-compile` runs without any issues.
- [ ] `make daemon-codestyle` runs without any issues.
- [ ] `make daemon-unittests` runs without any issues.
- [ ] `make webui-codestyle` runs without any issues. 
- [x] I understand and accept the [AGPL-3.0 license](LICENSE) and [code of conduct](CODE_OF_CONDUCT.md), and my contributions fall under these.

### CODE REVIEW

**Comment by jamesread** [comment]
Hey @matthewstrasiotto , thanks for the time spent in doing this - much appreciated :-) I've not had much time to work on OliveTin recently, so any contribution from others is always welcome. 

I'd like to see a screenshot of what the page looks like by default - a default unstyled H1 element might look pretty awful. My main workstation is powered off at the moment, but I'll take a look in the morning :-)

File: webui/main.js [hunk 1]
```diff
@@ -64,6 +64,12 @@ function processWebuiSettingsJson (settings) {
 
   document.querySelector('#section-switcher').hidden = !settings.ShowNavigation
   document.querySelector('footer[title="footer"]').hidden = !settings.ShowFooter
+
+  if (settings.pageTitle) {
+    document.title = setttings.pageTitle;
```
Comment (jonny7read) [inline webui/main.js]: Looks like you've got a typo here on `setttings`, should be `settings`
```suggestion
    document.title = settings.pageTitle;
```

**Review by jonny7read — CHANGES_REQUESTED** [review]
Typo

**Comment by jamesread** [comment]
Hey @strazto , this is long overdue, sorry for the late review. Life has just been busy!

I just did a quick merge commit to bring this up to date with main, and built it, and this is the screenshot I was curious about;

image

I obviously would like to make this prettier! I have been working on a new sidebar feature as part of the UI update / entity support coming soon, and think the "title" fits pretty nicely in the "sidebar". 

image

I'm not sure if the title should be an overall "global" title, or if it should show the current "dashboard title". Need to give that some thought.

**Review by jamesread — APPROVED** [review]
LGTM! Thanks @strazto !

**Pull request merged**